### DISCUSSION TITLE
is it possible run JetStream and nats-streaming

### DISCUSSION THREAD

**Discussion by alifpay** [nats-io/k8s#371] [Q&A]
we have nats-streaming server and version of nats-server 2.3.3 

can we active JetStream?

or setup new nat-server?

**Comment by wallyqs** [comment]
It is possible since they are in different namespaces, nats-streaming runs under `_STAN.>` global namespace whereas nats-server jetstream is just NATS subjects.  It would only break if you create a subject named `_STAN.foo` to be used with JetStream.  For JetStream usage I recommend starting with latest nats-server v2.6.3  and Go client v1.13.0 since would give the most stable setup.
